test/game_test.rb

```diff
@@ -0,0 +1,23 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/contestant"
+require "./lib/game"
+
+class GameTest < Minitest::Test
+
+  def setup
+    @pick_4 = Game.new('Pick 4', 2, false)
```

Comment:
Remember to stick to the interaction pattern, because if you change it, it will hurt your ability to implement from that point on as everything is set up very intentionally.